src/test/java/io/managed/services/test/kafka/KafkaMgmtAPITest.java

```diff
@@ -87,8 +87,6 @@ public class KafkaMgmtAPITest extends TestBase {
         "kafka_controller_kafkacontroller_offline_partitions_count",
         "kafka_controller_kafkacontroller_global_partition_count",
         "kafka_topic:kafka_log_log_size:sum",
-        "kafka_namespace:haproxy_server_bytes_in_total:rate5m",
```

Comment:
what's this part of the change for?

Reply:
we haven't renamed metrics that are part of the public API as part of this change.

Reply:
+1

Reply:
thanks for insight, will remove this change and keep only removing in billing. For now testing of metric's federation will fail as is discussed in new issue.